packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -15,6 +15,8 @@ import {readJson} from 'fs-extra'
 import {Minimatch} from 'minimatch'
 import git from 'git-rev-sync'
 import validator from 'validator'
+import {execSync} from 'child_process'
+import semver from 'semver'
```

Comment:
Make sure you add `semver` as a dependency! (This currently works because it's present as a _nested_ dependency, but you can't rely on those to always be there.)

Reply:
good catch will add!